localstripe/resources.py

```diff
@@ -244,6 +244,67 @@ def do_expand(path, obj):
         return obj
 
 
+class Balance(object):
+    object = 'balance'
+
+    def __init__(self, **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
```

Comment:
If this object cannot be created nor update (no `PUT` nor `POST`), then `__init__()` will never be called with keyword arguments. You can remove it and drop lines 251-252.